.github/workflows/codeql.yml

```diff
@@ -0,0 +1,97 @@
+name: "CodeQL Advanced"
+
+on:
+  push:
+  pull_request:
+  merge_group:
+  schedule: # Trigger a job on default branch at 4AM PST everyday
+    - cron: 0 11 * * *
+
+jobs:
+  analyze:
```

Comment:
This job should be marked as `required` in the repo configuration before merging this PR.